buildconfig/config_emsdk.py

```diff
@@ -78,7 +78,7 @@ def __init__(self, name, envname, exename, minver, defaultlibs, version_flag="--
 
 
         except (ValueError, TypeError):
-            print ('WARNING: "%s" failed!' % command)
+            print (f'WARNING: "{command}" failed!')
```

Comment:
Maybe someone in the future should create a PR fixing all of the `print ()`s to `print()`

Reply:
Yes.   Separate PR.